CMakeLists.txt

```diff
@@ -258,65 +258,70 @@ if (NOT CMAKE_BUILD_TYPE)
 endif (NOT CMAKE_BUILD_TYPE)
 
 if (NOT (WIN32 OR CYGWIN))
-  option (BUILD_SHARED_LIBS "Build shared libraries." ON)
+  option (SC_BUILD_SHARED_LIBS "Build shared libraries." ON)
 else (NOT (WIN32 OR CYGWIN))
-  option (BUILD_SHARED_LIBS "Build shared libraries." OFF)
+  option (SC_BUILD_SHARED_LIBS "Build shared libraries." OFF)
 endif (NOT (WIN32 OR CYGWIN))
-if (BUILD_SHARED_LIBS AND (WIN32 OR CYGWIN))
-  message (WARNING "The compilation of SystemC as a DLL on Windows is currently not supported!")
-  set (BUILD_SHARED_LIBS CACHE BOOL "Build shared libraries." OFF FORCE)
-endif (BUILD_SHARED_LIBS AND (WIN32 OR CYGWIN))
 
-option (BUILD_SOURCE_DOCUMENTATION "Build source documentation with Doxygen." OFF)
+set (SC_LIB_TYPE STATIC)
+if (SC_BUILD_SHARED_LIBS OR BUILD_SHARED_LIBS)
+  if (WIN32 OR CYGWIN)
+    message (FATAL_ERROR "The compilation of SystemC as a DLL on Windows is currently not supported!")
```

Comment:
Is this still relevant since https://github.com/OSCI-WG/systemc/pull/597 got merged?